.LibreSprite.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
It would be better to reuse https://github.com/LibreSprite/LibreSprite/blob/master/desktop/libresprite.desktop instead of duplicating it as a hidden file.